pyproject.toml

```diff
@@ -0,0 +1,62 @@
+# pyproject.toml
+# See https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html
+# for more information about the pyproject.toml file.
+
+# Recommended Packaging Guide
+# Run:
+# 1. `pip install --upgrade build` to install the build package.
+# 2. `python -m build` to build the package.
+# 3. `twine upload dist/*` to upload the package to PyPI.
+
+[build-system]
+requires = ["setuptools>=60"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "truelearn"
+dynamic = ["version"]
+description = "A Machine learning library for recommending open educational resources"
+readme = "README.rst"
+requires-python = ">=3.7"
+keywords = ["machine learning", "python", "statistics", "probability"]
+license = {file = "LICENSE"}
+authors = [
+    {name = "Sahanbull"},
+    {name = "KD-7"},
+    {name = "yuxqiu"},
+    {name = "deniselezi"},
+    {name = "aaneelshalman"},
+]
+classifiers = [
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Topic :: Scientific/Engineering :: Artificial Intelligence",
+    "Topic :: Software Development :: Libraries",
+    "Topic :: Software Development :: Libraries :: Python Module",
+]
+dependencies = [
+    'numpy>=1.14.1',
+    'trueskill>=0.4.5',
+    'mpmath>=1.1.0',
+    'scikit-learn>=0.19.1',
+]
+
+[tool.setuptools.dynamic]
+version = {attr = "truelearn.__version__"}
+
+[tool.setuptools.packages.find]
+where = ["."]
+include = ["truelearn*"]
```

Comment:
The `*` is unnecessary.

Reply:
Fixed in fcf63c50999bcb98299d60aa4ae6fce40cb6e0f6